cmd/levityd/main.go

```diff
@@ -50,11 +58,45 @@ func expandPaths() error {
 	if err != nil {
 		return err
 	}
+
 	privateKeyPath = s
 
 	return nil
 }
 
+func initTLS() (grpc.ServerOption, error) {
+
+	log.Printf("Loading TLS certificate from %s", certificatePath)
+	log.Printf("Loading private key from %s", privateKeyPath)
+	log.Printf("Client CA Cert path: %s", clientCACertPath)
+
+	idCert, err := tls.LoadX509KeyPair(certificatePath, privateKeyPath)
+	if err != nil {
+		return nil, err
+	}
+
+	tlsCfg := &tls.Config{
```

Comment:
Design document says you'll only support TLS 1.3, but I think this supports TLS 1.2 and 1.3.

Reply:
Added minimum version to TLS config, and integration test to assert it is enforced.